example/register_device/lib/ThingooConnector/ThingooConnector.h

```diff
@@ -0,0 +1,44 @@
+/***************************************************************
+   esp8266Thingoo is a library for the ESP8266/Arduino platform
+   to communicate with Thingoo API
+   Built for KNI Project "Thingoo" https://github.com/ThingooKNI/
+ **************************************************************/
+
+#ifndef ThingooConnector_h
+#define ThingooConnector_h
+
+#include <Arduino.h>
```

Comment:
These headers shouldn't be included in .h file since they're not required here, apart from `ArduinoJson` and `Arduino`